docs/development/workflow/command_history.sh

```diff
@@ -22,20 +22,19 @@ git push --set-upstream mwcraig i-1761 # Hey GitHub, get ready to receive my wor
 #
 conda create -n apy-1761 --clone root   # copy my default environment
 source activate apy-1761                # switch to this environment
-# next step DOES NOT WORK in python 3
-python setup.py develop                 # install astropy
+pip install -e .                        # install astropy
 #
 # Check setup by running tests
 #
 cd astropy/coordinates/tests    # get ready to run coordinate tests
-py.test                         # test
+pytest                          # test
 ls                              # what is here?
 #
 # Write a test in test_arrays.py to expose this bug
 #
 # After edit, re-test
 #
-py.test test_arrays.py          # Hopefully this succeeds
+pytest test_arrays.py           # Hopefully this succeeds
```

Comment:
``pytest astropy/coordinates/tests/test_arrays`` - etc for the rest of the instructions

Reply:
I think updating the actual instruction is out of scope here. The whole thing is way outdated. The log shows Python 2, etc etc. Maybe @mwcraig can update it when he has time. I am not even sure where this info is displayed. I just modify it for consistency in regards to "py.test" since `grep` found it.

Reply:
I had completely forgotten about this. It probably needs to be re-written from the ground up if we want it to stay around. I agree this is outside the scope of this PR.

Reply:
Okay, we can follow-up over at #10777 . Thanks!